oak_runtime/src/node/http/server.rs

```diff
@@ -555,6 +555,7 @@ impl HttpResponseReceiver {
         let response_receiver = crate::io::Receiver::<HttpResponse>::new(ReadHandle {
             handle: self.response_reader,
         });
+        info!("Trying to read the response...");
```

Comment:
Just checking: are these logs intentional?

Reply:
This was intentional, but perhaps not quite consistent with the rest of the logs. Removed.